src/Microsoft.ML.FastTree/TreeTrainersCatalog.cs

```diff
@@ -136,7 +136,7 @@ public static class TreeExtensions
         }
 
         /// <summary>
-        /// Predict a target using generalized additive models trained with the <see cref="BinaryClassificationGamTrainer"/>.
+        /// Predict a target using generalized additive models trained with the <see cref="GamBinaryClassificationTrainer"/>.
```

Comment:
I wonder if we should connect the acronyms in the doc:

`Predict a target using generalized additive models (GAM) trained with the <see cref="GamBinaryClassificationTrainer"/>.`

Reply:
Thanks for the suggestion. Will fix.